openshift/templates/api/config/dev/default.json

```diff
@@ -39,7 +39,7 @@
     "multiUse": true
   },
   "authentication": {
-    "jwksUri": "https://oidc.gov.bc.ca/auth/realms/gzyg46lx/protocol/openid-connect/certs",
+    "jwksUri": "https://dev.oidc.gov.bc.ca/auth/realms/gzyg46lx/protocol/openid-connect/certs",
```

Comment:
![image](https://user-images.githubusercontent.com/10504350/114209524-8d5c5900-9913-11eb-9203-eeacf7253a25.png)

I see a few files renamed like this.  Was the renaming of `prod` to `dev` intentional?  What happened to the `prod` version?

Was the removal of the `bc` profile intentional?
![image](https://user-images.githubusercontent.com/10504350/114209761-dad8c600-9913-11eb-8fd7-215b11ee6494.png)

Reply:
prod is still there. what I think might have happened is that when I was removing the bc folder, git must have gotten mixed up and tracked changes across the wrong file? ie `config/bc/prod/default.json -> config/dev/default.json` instead of `config/bc/prod/default.json -> config/prod/default.json`.

Reply:
It might have been my faiult - I had started working on this and handed it off to Wade, and he inherited my half-baked config files (we did not have a dev environment until OCP4, I must have copied files over and did not get to update them).